src/main/java/lz/modelTesting/repositories/MockupsRepository.java

```diff
@@ -0,0 +1,14 @@
+package lz.modelTesting.repositories;
+
+import lz.modelTesting.documents.Mockup;
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.data.mongodb.repository.Query;
+
+import java.util.List;
+
+public interface MockupsRepository extends MongoRepository<Mockup,Integer> {
+
+    @Query("{ 'name' : ?0 }")
```

Comment:
this query is unnecessary. Spring is clever enough to resolve findBySomething
see: https://www.baeldung.com/queries-in-spring-data-mongodb

Reply:
Deleted